apps/web/src/components/common/ImageManager/TypeA/TypeA.tsx

```diff
@@ -0,0 +1,128 @@
+'use client';
+
+import { Icon } from '../../../../../../../packages/ui/src/components/Icon';
+import { Text } from '../../../../../../../packages/ui/src/components/Text';
```

Comment:
p1;
apps/web은 절대경로 인식이 되니까 절대경로로 작성하면 될 것 같아요!
```suggestion
import { Icon } from '@repo/ui/Icon';
import { Text } from '@repo/ui/Text';
```

Reply:
이거 packages/ui 에 넣었다가 다시 옮기고 수정을 놓쳤네요.. 꼼꼼한 리뷰 감사해요!

8612a8b05b9f792a959367c20df601ee1d5e47c7